dmutils/formats.py

```diff
@@ -47,6 +94,17 @@ def utcdatetimeformat(value, default_value=None):
 
 
 def datetodatetimeformat(value):
```

Comment:
Interesting. This method feels like it has the wrong name. It doesn't take a date (`date to`) and turn it into the same format as `datetimeformat`. It takes a date and turns it into the same format as comes out of our `dateformat`. Breaking change, but it feels like this should be called `datetodateformat`...

Reply:
We should also probably rename `utcdatetimeformat` to make it clear that we're *forcing* to UTC, but probably not in this PR?

Reply:
Yeah, seems reasonable. But not for this PR as you say. 👍